project/plugins.sbt

```diff
@@ -1,3 +1,6 @@
 addSbtPlugin("org.scalameta" % "sbt-scalafmt"      % "2.3.0")
-addSbtPlugin("org.scoverage" % "sbt-scoverage"     % "1.9.3")
+addSbtPlugin("org.scoverage" % "sbt-scoverage"     % "2.0.7")
```

Comment:
Fixes this issue
<img width="1339" alt="image" src="https://user-images.githubusercontent.com/356718/227343145-a59c5ba6-acc8-4778-ac83-31afd284520a.png">